src/core/index.ts

```diff
@@ -93,57 +99,117 @@ export interface PromptUiActions {
     };
 }
 
+export interface UnderlayConfig {
```

Comment:
Wonder do we have a naming consistency weirdness brewing here?

The code calls it UnderlayConfig, the UI calls it 'backdrop' IIRC, and the CSS calls it background mask.

Worth lining all up prior to shipping?

Reply:
Could do... could just make a sweeping change from portal to SDK and just rename it to `backgroundMask` throughout

Reply:
Hah this is making my head spin, is it back _drop_ or back _ground_ ? CSS says `kumulos-background-mask` atm, does UI say different? :D

I'm +1 for lining up at this point if we can, just seems like it'll cause less confusion in future.

Reply:
Actually, we settled on `Mask website when prompt is shown` for UI so will just align config + refs to `backgroundMask` not backdrop